src/component-library/Select/Select.style.tsx

```diff
@@ -68,8 +68,7 @@ const StyledTriggerValue = styled(Span)<StyledTriggerValueProps>`
 
 const StyledList = styled(List)`
   overflow: auto;
-  padding: 0 ${theme.dialog.medium.body.paddingX} ${theme.dialog.medium.body.paddingY}
-    ${theme.dialog.medium.body.paddingX};
+  padding: 0 ${theme.dialog.medium.body.paddingX} ${theme.dialog.medium.body.paddingX};
```

Comment:
This is slightly wrong in that we're applying the paddingX value to the bottom. Not sure if it's worth changing the theme to support this.

Reply:
Yeah, I get your point. But this way you make sure that the bottom as the same padding as the sides.

Reply:
And this is only one scenario. If things like this appear later on, lets address this

Reply:
Yeah I'm fine with it—something we might want to fix if we're tidying up the theme, but not worth changing now.